recipes.adoc

```diff
@@ -11,11 +11,35 @@ and BRS-B (for "Bespoke").
 
 === BRS-I Recipe
 
-To simplify software testing and distribution, many operating system
-providers wish to build a single ACPI-based operating system image
-that can be booted and run on platforms supporting a common
-specification base.  The BRS-I recipe documents a standard runtime
-firmware interface to facilitate this.
+The BRS-I recipe aims to simplify end-user experiences, software
```

Comment:
end-user experiences - Is that necessary ? Given that the BRS-I cares about successful boot only, the user experience may be completely different one system to another.

Reply:
e.g The graphics/network example you gave above results in completely different end user experience.

Reply:
BRS-I simplifies the experience because it makes it trivial to install off-the-shelf OS on a variety of systems, same as is done today for x86 and AArch64 SystemReady SR/ES systems. That's what is meant here.

It's an important goal. If the spec is so weak that it means the same level of "involvement" to figure out how to find/build a bootable/working OS image, we will have failed.

Reply:
It makes sense now. I misinterpreted the "end-user" experience earlier. The build/boot experience should be as boring as possible. Anybody with prior experience  with ARM64/x86 should be able to get it working in no time.